server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/view/cassandra/DeletedMailsDAOTest.java

```diff
@@ -37,7 +39,9 @@
     private static final MailKey MAIL_KEY_2 = MailKey.of("mailkey2");
 
     @RegisterExtension
-    static CassandraClusterExtension cassandraCluster = new CassandraClusterExtension(CassandraMailQueueViewModule.MODULE);
+    static CassandraClusterExtension cassandraCluster = new CassandraClusterExtension(CassandraModule.aggregateModules(
```

Comment:
how is it related ?

Reply:
You can't launch it individually (from your IDE) without that.

It should be drop from this PR IMO.